lib/osbuild.py

```diff
@@ -214,6 +214,12 @@ def add_package(self, name):
             name = "{name}"
         ''') + '\n'
 
+    def add_package_group(self, name):
+        self.assembled += util.dedent(fr'''
+            [[groups]]
+            name = "{name}"
+        ''') + '\n'
```

Comment:
[4.7. RHEL image builder blueprint format](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-command-line-interface_composing-a-customized-rhel-system-image#composer-blueprint-format_creating-system-images-with-composer-command-line-interface)
```
[[groups]]
name = "group-name"
```
> The group entry describes a group of packages to be installed into the image. 

And [4.8.2. Selecting a package group](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-command-line-interface_composing-a-customized-rhel-system-image#selecting-a-package-group_creating-system-images-with-composer-command-line-interface)
```
[[packages]]
name = "package_group_name"
```
>  Replace "package_group_name" with the name of the group. For example, "tmux". 

I don't see the difference. But just looking at function name and RHEL doc examples, the latter seems as the right one.

Reply:
I'll ask osbuild developers, maybe one of them is obsolete now.

Reply:
Turns out `[[groups]]` is the correct syntax (as I'm using it currently in the code), using groups in `[[packages]]` should not work and the documentation is probably wrong.